config/config.pubsub.minimal.hocon

```diff
@@ -11,7 +11,7 @@
     "host": "localhost"
     "database": "snowplow"
     "username": "postgres"
-    "password": ${POSTGRES_PASSWORD}
+    "password": "mysecretpassword"
```

Comment:
This part is changed since make the test pass. Otherwise, it was throwing config parsing error.

Reply:
[This line](https://github.com/snowplow-incubator/snowplow-postgres-loader/blob/master/project/BuildSettings.scala#L114) is supposed to ensure that the tests pass, by setting the environment variable in tests.  I would understand better about why you were getting a config parsing error in the tests, because it suggests that something is not working where previously it _was_ working.  Did you run the tests using sbt?

Reply:
I run the CliSpec test with Intellij's own test runner. I guess that was the reason why I was getting the error. I just tried to run it with sbt and it run without any problem. I will revert it back 👍